src/lib/components/chat/Settings/Models.svelte

```diff
@@ -589,7 +589,7 @@
 												on:change={() => {
 													console.log(modelInputFile);
 												}}
-												accept=".gguf"
+												accept=".gguf",".safetensors"
```

Comment:
Shouldn't this be `.bin` and not `.safetensors`?

Reply:
Both of them might work. Although don't rely on me as I am not too sure. I will have a look into it.

Reply:
If you could check I'd greatly appreciate it!

Reply:
Alright, let me know if there is anything else you need me to look at.